webpack/localization-plugin/src/loaders/SingleLocaleLoader.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { loader } from 'webpack';
+import * as loaderUtils from 'loader-utils';
+import * as jju from 'jju';
+
+import { ILocJsonFile } from '../interfaces';
+
+export interface ILocJsonLoaderOptions {
+  /**
+   * The outer map's keys are the resolved, uppercased file names.
+   * The inner map's keys are the string identifiers and its values are the string values.
+   */
+  resolvedStrings: Map<string, Map<string, string>>;
+
+  /**
+   * If set to `true`, use the passthrough locale
+   */
+  passthroughLocale: boolean;
+}
+
+export default function (this: loader.LoaderContext, content: string): string {
+  const {
+    resolvedStrings,
+    passthroughLocale
+  } = loaderUtils.getOptions(this) as ILocJsonLoaderOptions;
+  const locJsonFilePath: string = this.resourcePath;
+  const normalizedLocJsonFilePath: string = locJsonFilePath.toUpperCase();
```

Comment:
Any particular reason why UpperCase ?

Reply:
That was to do case insensitive path comparison. Doesn't seem to matter, though. Removed.